.github/workflows/ci-latest-packages.yml

```diff
@@ -0,0 +1,79 @@
+name: latest version of packages
```

Comment:
Currently, the daily CI is using the latest version of packages, and the regular CI is using locked version
I think we can keep the same logic here